### ISSUE TITLE
[BUG]/[Question] - `Get-PnPPlannerBucket` bucket ordering is  lost

### ISSUE THREAD

**Issue by milanholemans** [pnp/powershell#1918]
### Reporting an Issue or Missing Feature
In my perspective it's a bug, but it's also a question.
I'm retrieving Planner buckets via `Get-PnPPlannerBucket`. I noticed that the ordering is not the same as in the UI.
This is strange since Graph does maintain the order of the buckets when I retrieve them via Graph explorer.

When I looked at the code I noticed that after the buckets are fetched, a custom sorting is done on the orderHint (line 405).

https://github.com/pnp/powershell/blob/0bb321e5679369eba510c183039de72f0fa95e21/src/Commands/Utilities/PlannerUtility.cs#L402-L406

Since the orderHint is a weird value ([docs](https://docs.microsoft.com/en-us/graph/api/resources/planner-order-hint-format?view=graph-rest-1.0)) sorting it alphabetically is not the way to do it. The original sorting of the buckets are gone and I don't know anymore which bucket is first and which is last.

This is what I have in Planner
![image](https://user-images.githubusercontent.com/11723921/171065946-047738f7-a4cc-4583-879d-b15ed593cb16.png)

This is what I get when I use the Graph call:
![image](https://user-images.githubusercontent.com/11723921/171065893-b876e027-d90d-4eb3-a6c7-410460bdfc8e.png)

This is what is returned via PnP.PowerShell
![image](https://user-images.githubusercontent.com/11723921/171065927-77b3b5e4-d3ed-4733-98f4-b0f70ccfb144.png)

As you can see, Graph maintains the order (in reverse order). But PowerShell kind of shuffles them.

My question is fairly simple. Can I remove the custom sorting so the Graph sorting is maintained? 😃 

### Expected behavior 
I'd like the buckets to maintain their order.

### Actual behavior 
Buckets are sorted alphabetically on oderHint and thus sort of shuffled.

### Steps to reproduce behavior
1. Create a plan with 5 buckets.
2. Execute cmdlet `Get-PnPPlannerBucket`

### What is the version of the Cmdlet module you are running?
1.10.31-debug

### Which operating system/environment are you running PnP PowerShell on?
- [x] Windows
- [ ] Linux
- [ ] MacOS
- [ ] Azure Cloud Shell
- [ ] Azure Functions
- [ ] Other : please specify

**Comment by KoenZomers** [comment]
Thanks for doing all the awesome research here yourself already @milanholemans ! Saves us a lot of time. I agree that OrderHint to look like a weird beast. Can you confirm that if you would move the buckets around, Graph will also return them in the same changed order (but then reversed as you mentioned)? In other words, can we always rely on Graph to keep the right order if we totally ignore OrderHint? If yes, then please go ahead and PR removing that line.

**Comment by milanholemans** [comment]
Hi @KoenZomers, thanks for your reply.

I just made an empty plan with 10 buckets. I can confirm that when I drag the buckets around, Graph maintaining the (reversed) order. I also came across this blog that confirms this behavior: https://laurakokkarinen.com/how-to-sort-tasks-using-planner-order-hint-and-microsoft-graph

![image](https://user-images.githubusercontent.com/11723921/171291091-b8eea2b4-7929-42ec-a4ac-5af24cc46b0b.png)
